src/index.html

```diff
@@ -6,7 +6,40 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="calendar calendar--start-wed calendar--length-29 page__calendar">
+      <div class="calendar__item calendar__item--day-"></div>
```

Comment:
```suggestion
      <div class="calendar__item"></div>
```
В модификаторе дописываются стили, которые отличают конкретный элемент от подобных. Здесь же у каждого элемента одинаковый модификатор, в этом нет смысла